src/screen/AccountsScreen.tsx

```diff
@@ -189,28 +179,70 @@ function AccountItem({
   const theme = useTheme();
 
   return (
-    <ListItem
+    <List.Item
       onPress={onPress}
-      accessoryLeft={(p) => (
-        <View {...p}>
+      left={() => (
+        <View style={globalStyles.justifyCenter}>
           <Identicon value={String(identity.accountId)} size={25} />
         </View>
       )}
-      description={`${isExternal ? 'External' : ''}`}
-      title={(p) => (
-        <View {...p}>
+      title={() => (
+        <View style={globalStyles.justifyCenter}>
           <AccountInfoInlineTeaser identity={identity} />
+          {isExternal && <Caption>External</Caption>}
         </View>
       )}
-      accessoryRight={(p) => (
-        <TouchableOpacity {...p} onPress={toggleFavorite}>
-          <Icon
-            style={globalStyles.icon25}
-            fill={isFavorite ? theme['color-warning-500'] : theme['color-basic-500']}
-            name={isFavorite ? 'star' : 'star-outline'}
-          />
-        </TouchableOpacity>
+      right={() => (
+        <IconButton
+          onPress={toggleFavorite}
+          color={isFavorite ? theme.colors.accent : theme.colors.disabled}
+          icon={isFavorite ? 'star' : 'star-outline'}
+        />
       )}
     />
   );
 }
+
+const Buttons = ({navigation}: {navigation: Props['navigation']}) => {
+  const [state, setState] = React.useState({open: false});
+
+  const onStateChange = ({open}: {open: boolean}) => setState({open});
+
+  const {open} = state;
+
+  return (
+    <Provider>
+      <Portal>
```

Comment:
Does it work properly?
The app should already be wrapped with Provider in themeContext

Reply:
yes, without this provider the FAB.Group would show up over the whole app ( all navigation screens ) and with this Provider here it would be just inside this page.

Reply:
Probably provider seems to be working like that but there's still a problem. It is taking up screen space.
The problem is visible is the list is long.

<img src=https://user-images.githubusercontent.com/16683923/143257633-8275c7de-0a94-4721-ba99-e2f78af51d1d.png width=250 />

Reply:
yes thanks for noticing, I wrapped the provider around the whole page and that fixed it.